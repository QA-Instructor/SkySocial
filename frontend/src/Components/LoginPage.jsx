import React from 'react'
import LoginForm from "./LoginForm"
import "../resources/loginPage.css"
export const LoginPage = () => {
  return (
    <div className='loginPage'>
        <div className='loginPage-logo-container'>
            <div className='loginPage-logo'>
                Sky Social
            </div>
            <div className='loginPage-info'>
                Sky Social is your gateway to a social life at Sky.
                Explore opportunities to get involved in sports and 
                various socials around Sky.
            </div>
        </div>
        <div className='loginPage-form-container'>
            <LoginForm/>
        </div>
        
    </div>
  )
}

export default LoginPage