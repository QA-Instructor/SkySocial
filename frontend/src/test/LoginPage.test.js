import React from 'react';
import { create } from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';


import LoginPage from '../Components/LoginPage';

describe('Login Page Snapshot tests',()=>{
    test(`Login Page matches snapshot`, () => {
        const header = create(LoginPage);
        expect(header.toJSON()).toMatchSnapshot();
    });
});