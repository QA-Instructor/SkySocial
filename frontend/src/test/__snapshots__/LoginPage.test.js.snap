// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot tests Header matches snapshot 1`] = `null`;

exports[`Login Page Snapshot tests Login Page matches snapshot 1`] = `null`;
